From 5b4aad3123992badf7f1bfb4c23bff390b9c9494 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Wed, 4 Sep 2024 14:01:58 +0000
Subject: [PATCH] Fix JSON validation

json-glib 1.10 is more strict when parsing the JSON data.

Closes: https://gitlab.gnome.org/GNOME/cheese/-/issues/183
---
 data/cheese-viewport.json | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/data/cheese-viewport.json b/data/cheese-viewport.json
index cf03fcec..132b68d2 100644
--- a/data/cheese-viewport.json
+++ b/data/cheese-viewport.json
@@ -53,9 +53,9 @@
   "type": "ClutterActor",
   "children":
   [
-    'video_preview',
-    'countdown_layer',
-    'error_layer'
+    "video_preview",
+    "countdown_layer",
+    "error_layer"
   ]
 },
 {
-- 
GitLab

